index.js

```diff
@@ -1,5 +1,7 @@
+const { kMaxLength } = require('buffer');
```

Comment:
This import is unused

Reply:
You wanted me to delete this unused import correct? 

I didn't want to assume since this is my first time working on this level of a pull request 😄

Reply:
Yep delete it. Sorry should've been more descriptive.